### ISSUE TITLE
使用en_PP-OCRv3_rec_train预训练模型进行识别

### ISSUE THREAD

**Issue by chfdeng** [PaddlePaddle/PaddleOCR#6935]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：Ubuntu18.04
- 版本号/Version：Paddle：paddlepaddle-gpu==2.3.1.post116   PaddleOCR：2.3.1 问题相关组件/Related components：
- 运行指令/Command Code：python3 tools/infer_rec.py -c configs/rec/PP-OCRv3/en_PP-OCRv3_rec.yml -o Global.pretained_model=./pretrain_models/en_PP-OCRv3_rec_train/best_accuracy 
- 完整报错/Complete Error Message：
[2022/07/19 16:17:37] ppocr INFO: Architecture : 
[2022/07/19 16:17:37] ppocr INFO:     Backbone : 
[2022/07/19 16:17:37] ppocr INFO:         last_conv_stride : [1, 2]
[2022/07/19 16:17:37] ppocr INFO:         last_pool_type : avg
[2022/07/19 16:17:37] ppocr INFO:         name : MobileNetV1Enhance
[2022/07/19 16:17:37] ppocr INFO:         scale : 0.5
[2022/07/19 16:17:37] ppocr INFO:     Head : 
[2022/07/19 16:17:37] ppocr INFO:         head_list : 
[2022/07/19 16:17:37] ppocr INFO:             CTCHead : 
[2022/07/19 16:17:37] ppocr INFO:                 Head : 
[2022/07/19 16:17:37] ppocr INFO:                     fc_decay : 1e-05
[2022/07/19 16:17:37] ppocr INFO:                 Neck : 
[2022/07/19 16:17:37] ppocr INFO:                     depth : 2
[2022/07/19 16:17:37] ppocr INFO:                     dims : 64
[2022/07/19 16:17:37] ppocr INFO:                     hidden_dims : 120
[2022/07/19 16:17:37] ppocr INFO:                     name : svtr
[2022/07/19 16:17:37] ppocr INFO:                     use_guide : True
[2022/07/19 16:17:37] ppocr INFO:             SARHead : 
[2022/07/19 16:17:37] ppocr INFO:                 enc_dim : 512
[2022/07/19 16:17:37] ppocr INFO:                 max_text_length : 25
[2022/07/19 16:17:37] ppocr INFO:         name : MultiHead
[2022/07/19 16:17:37] ppocr INFO:     Transform : None
[2022/07/19 16:17:37] ppocr INFO:     algorithm : SVTR
[2022/07/19 16:17:37] ppocr INFO:     model_type : rec
[2022/07/19 16:17:37] ppocr INFO: Eval : 
[2022/07/19 16:17:37] ppocr INFO:     dataset : 
[2022/07/19 16:17:37] ppocr INFO:         data_dir : ./train_data
[2022/07/19 16:17:37] ppocr INFO:         label_file_list : ['./train_data/val_list.txt']
[2022/07/19 16:17:37] ppocr INFO:         name : SimpleDataSet
[2022/07/19 16:17:37] ppocr INFO:         transforms : 
[2022/07/19 16:17:37] ppocr INFO:             DecodeImage : 
[2022/07/19 16:17:37] ppocr INFO:                 channel_first : False
[2022/07/19 16:17:37] ppocr INFO:                 img_mode : BGR
[2022/07/19 16:17:37] ppocr INFO:             MultiLabelEncode : None
[2022/07/19 16:17:37] ppocr INFO:             RecResizeImg : 
[2022/07/19 16:17:37] ppocr INFO:                 image_shape : [3, 48, 320]
[2022/07/19 16:17:37] ppocr INFO:             KeepKeys : 
[2022/07/19 16:17:37] ppocr INFO:                 keep_keys : ['image', 'label_ctc', 'label_sar', 'length', 'valid_ratio']
[2022/07/19 16:17:37] ppocr INFO:     loader : 
[2022/07/19 16:17:37] ppocr INFO:         batch_size_per_card : 128
[2022/07/19 16:17:37] ppocr INFO:         drop_last : False
[2022/07/19 16:17:37] ppocr INFO:         num_workers : 4
[2022/07/19 16:17:37] ppocr INFO:         shuffle : False
[2022/07/19 16:17:37] ppocr INFO: Global : 
[2022/07/19 16:17:37] ppocr INFO:     cal_metric_during_train : True
[2022/07/19 16:17:37] ppocr INFO:     character_dict_path : ppocr/utils/en_dict.txt
[2022/07/19 16:17:37] ppocr INFO:     checkpoints : None
[2022/07/19 16:17:37] ppocr INFO:     debug : False
[2022/07/19 16:17:37] ppocr INFO:     distributed : False
[2022/07/19 16:17:37] ppocr INFO:     epoch_num : 500
[2022/07/19 16:17:37] ppocr INFO:     eval_batch_step : [0, 2000]
[2022/07/19 16:17:37] ppocr INFO:     infer_img : doc/imgs_words/en/word_1.png
[2022/07/19 16:17:37] ppocr INFO:     infer_mode : False
[2022/07/19 16:17:37] ppocr INFO:     log_smooth_window : 20
[2022/07/19 16:17:37] ppocr INFO:     max_text_length : 25
[2022/07/19 16:17:37] ppocr INFO:     pretained_model : ./pretrain_models/en_PP-OCRv3_rec_train/best_accuracy
[2022/07/19 16:17:37] ppocr INFO:     pretrained_model : None
[2022/07/19 16:17:37] ppocr INFO:     print_batch_step : 10
[2022/07/19 16:17:37] ppocr INFO:     save_epoch_step : 3
[2022/07/19 16:17:37] ppocr INFO:     save_inference_dir : None
[2022/07/19 16:17:37] ppocr INFO:     save_model_dir : ./output/v3_en_mobile
[2022/07/19 16:17:37] ppocr INFO:     save_res_path : ./output/rec/predicts_ppocrv3_en.txt
[2022/07/19 16:17:37] ppocr INFO:     use_gpu : True
[2022/07/19 16:17:37] ppocr INFO:     use_space_char : True
[2022/07/19 16:17:37] ppocr INFO:     use_visualdl : False
[2022/07/19 16:17:37] ppocr INFO: Loss : 
[2022/07/19 16:17:37] ppocr INFO:     loss_config_list : 
[2022/07/19 16:17:37] ppocr INFO:         CTCLoss : None
[2022/07/19 16:17:37] ppocr INFO:         SARLoss : None
[2022/07/19 16:17:37] ppocr INFO:     name : MultiLoss
[2022/07/19 16:17:37] ppocr INFO: Metric : 
[2022/07/19 16:17:37] ppocr INFO:     ignore_space : False
[2022/07/19 16:17:37] ppocr INFO:     main_indicator : acc
[2022/07/19 16:17:37] ppocr INFO:     name : RecMetric
[2022/07/19 16:17:37] ppocr INFO: Optimizer : 
[2022/07/19 16:17:37] ppocr INFO:     beta1 : 0.9
[2022/07/19 16:17:37] ppocr INFO:     beta2 : 0.999
[2022/07/19 16:17:37] ppocr INFO:     lr : 
[2022/07/19 16:17:37] ppocr INFO:         learning_rate : 0.001
[2022/07/19 16:17:37] ppocr INFO:         name : Cosine
[2022/07/19 16:17:37] ppocr INFO:         warmup_epoch : 5
[2022/07/19 16:17:37] ppocr INFO:     name : Adam
[2022/07/19 16:17:37] ppocr INFO:     regularizer : 
[2022/07/19 16:17:37] ppocr INFO:         factor : 3e-05
[2022/07/19 16:17:37] ppocr INFO:         name : L2
[2022/07/19 16:17:37] ppocr INFO: PostProcess : 
[2022/07/19 16:17:37] ppocr INFO:     name : CTCLabelDecode
[2022/07/19 16:17:37] ppocr INFO: Train : 
[2022/07/19 16:17:37] ppocr INFO:     dataset : 
[2022/07/19 16:17:37] ppocr INFO:         data_dir : ./train_data/
[2022/07/19 16:17:37] ppocr INFO:         ext_op_transform_idx : 1
[2022/07/19 16:17:37] ppocr INFO:         label_file_list : ['./train_data/train_list.txt']
[2022/07/19 16:17:37] ppocr INFO:         name : SimpleDataSet
[2022/07/19 16:17:37] ppocr INFO:         transforms : 
[2022/07/19 16:17:37] ppocr INFO:             DecodeImage : 
[2022/07/19 16:17:37] ppocr INFO:                 channel_first : False
[2022/07/19 16:17:37] ppocr INFO:                 img_mode : BGR
[2022/07/19 16:17:37] ppocr INFO:             RecConAug : 
[2022/07/19 16:17:37] ppocr INFO:                 ext_data_num : 2
[2022/07/19 16:17:37] ppocr INFO:                 image_shape : [48, 320, 3]
[2022/07/19 16:17:37] ppocr INFO:                 prob : 0.5
[2022/07/19 16:17:37] ppocr INFO:             RecAug : None
[2022/07/19 16:17:37] ppocr INFO:             MultiLabelEncode : None
[2022/07/19 16:17:37] ppocr INFO:             RecResizeImg : 
[2022/07/19 16:17:37] ppocr INFO:                 image_shape : [3, 48, 320]
[2022/07/19 16:17:37] ppocr INFO:             KeepKeys : 
[2022/07/19 16:17:37] ppocr INFO:                 keep_keys : ['image', 'label_ctc', 'label_sar', 'length', 'valid_ratio']
[2022/07/19 16:17:37] ppocr INFO:     loader : 
[2022/07/19 16:17:37] ppocr INFO:         batch_size_per_card : 128
[2022/07/19 16:17:37] ppocr INFO:         drop_last : True
[2022/07/19 16:17:37] ppocr INFO:         num_workers : 4
[2022/07/19 16:17:37] ppocr INFO:         shuffle : True
[2022/07/19 16:17:37] ppocr INFO: profiler_options : None
[2022/07/19 16:17:37] ppocr INFO: train with paddle 2.3.1 and device Place(gpu:0)
W0719 16:17:37.017343 17667 gpu_resources.cc:61] Please NOTE: device: 0, GPU Compute Capability: 8.6, Driver API Version: 11.6, Runtime API Version: 11.6
W0719 16:17:37.020885 17667 gpu_resources.cc:91] device: 0, cuDNN Version: 8.4.
[2022/07/19 16:17:37] ppocr INFO: train from scratch
[2022/07/19 16:17:37] ppocr INFO: infer_img: doc/imgs_words/en/word_1.png
[2022/07/19 16:17:38] ppocr INFO: 	 result: #v	0.012823706492781639
[2022/07/19 16:17:38] ppocr INFO: success!

请教下，使用en_PP-OCRv3_rec_train预训练模型进行识别为什么结果都是错的呢
